### ISSUE TITLE
Repository ownership should have been moved, not as fork

### ISSUE THREAD

**Issue by esuomi** [docker-flow/docker-flow-proxy#12]
The main issue is that the repository source code is not searchable. To quote [GitHub: Searching in Forks](https://help.github.com/articles/searching-in-forks/):
> Forks are only indexed for code search when they have more stars than the parent repository. You will not be able to search the code in a fork that has less stars than its parent.

As it stands, this repo will be without code search for a _long_ time.

More reading about how's and why's: [Transferring a repository owned by your personal account](https://help.github.com/articles/transferring-a-repository-owned-by-your-personal-account/)

**Comment by thomasjpfan** [comment]
@vfarcic I think this is a good idea, for DFP and DFSL.

**Comment by vfarcic** [comment]
You're right. I made a mistake by forking the code. I'll work on this later this week.

**Comment by vfarcic** [comment]
Made a request. I should (hopefully) be done soon.
